GUI/sanity-test-halo/jest-puppeteer/jest.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    preset: "jest-puppeteer",
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/Seagate/cortx-experiments/pullRequest?prid=9333958)
```suggestion
  preset: "jest-puppeteer",
```